unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/GraphicCardWarningHUD/GraphicCardWarningHUDController.cs

```diff
@@ -1,8 +1,19 @@
+using DCL.Interface;
+using DCL.NotificationModel;
+using UnityEngine;
+
 public class GraphicCardWarningHUDController : IHUD
 {
-    private const string GRAPHIC_CARD_MESSAGE = "Your machine is not using a dedicated graphics card to run Decentraland. This might lead to performance issues. Check your browser and OS configuration.";
+    private const string GRAPHIC_CARD_MESSAGE = "Your machine is not using a dedicated graphics card to run Decentraland. This might lead to performance issues. Check your browser and OS configuration";
+    private readonly string restartMessage;
```

Comment:
I wonder why it should be an instance field and not a static one? Can't you move all the logic to the `static readonly` inline assignment?
The static initializer is called when the class is accessed for the first time so effectively it will be the same moment when the instance constructor is called now.